### PR TITLE
Adds glowy eyes to badbois and sec/phoron oties.

### PR DESCRIPTION
**Pull request by Verkister**
-Not glowing when resting though because of reasons.
spookyboye2e5576e111b9966b42161c867b3404a2bb232645b18abef19ee34a5bc8e4ffae

### CODE REVIEW

**Comment by Very-Soft** [comment]
Glowbois!

File: code/modules/mob/living/simple_animal/vore/badboi.dm [hunk 1]
```diff
@@ -87,7 +88,29 @@
 		return EatTarget()
 	else ..()
 
+/mob/living/simple_animal/hostile/badboi/proc/add_eyes()
+	if(!eye_layer)
+		eye_layer = image(icon, "badboi-eyes")
+		eye_layer.plane = PLANE_LIGHTING_ABOVE
+		modifier_overlay = eye_layer
+	overlays += eye_layer
```
Comment (Arokha) [inline code/modules/mob/living/simple_animal/vore/badboi.dm]: `add_overlays(eye_layer)` and remove the `modifier_overlay = eye_layer`

File: code/modules/mob/living/simple_animal/vore/badboi.dm [hunk 2]
```diff
@@ -87,7 +88,29 @@
...
+
+/mob/living/simple_animal/hostile/badboi/proc/remove_eyes()
+	overlays -= eye_layer
```
Comment (Arokha) [inline code/modules/mob/living/simple_animal/vore/badboi.dm]: `cut_overlay(eye_layer)`

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 3]
```diff
@@ -315,7 +326,32 @@
 		else
 			..()
 
+/mob/living/simple_animal/otie/proc/add_eyes()
+	if(!eye_layer)
+		eye_layer = image(icon, "[eyetype]-eyes")
+		eye_layer.plane = PLANE_LIGHTING_ABOVE
+		modifier_overlay = eye_layer
+	overlays += eye_layer
```
Comment (Arokha) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: `add_overlay(eye_layer)`, and drop the modifier_overlay thing

File: code/modules/mob/living/simple_animal/vore/otie.dm [hunk 4]
```diff
@@ -315,7 +326,32 @@
...
+
+/mob/living/simple_animal/otie/proc/remove_eyes()
+	overlays -= eye_layer
```
Comment (Arokha) [inline code/modules/mob/living/simple_animal/vore/otie.dm]: `cut_overlay(eye_layer)`

**Comment by Verkister** [comment]
Probably https://github.com/PolarisSS13/Polaris/commit/e248b89b7b4a12f1a9df74781627c63febaef299 making the simplemob update_icon() replace the overlays with a list only containing the modifier_overlay and nothing else

**Pull request merged**